published_topics:
  flir_image_topic: flir_lepton/image/gray
  flir_image16_topic: flir_lepton/image/gray16
  flir_rgb_image_topic: flir_lepton/image/rgb
  flir_temper_topic: flir_lepton/temperatures
  flir_batch_topic: flir_lepton/batch
flir_urdf:
  camera_optical_frame: flir_lepton_optical_frame
iface:
  device_port: "/dev/spidev1.0"
  bits: 8
  speed: 20000000
  delay: 0
  packet_size: 164
  packets_per_frame: 60
frame_rate: 25
dataset_file_name: "dataset_spline"
